StortSpelProjekt/Engine/src/Physics/Physics.h

```diff
@@ -20,23 +35,32 @@ class Physics
 	// Add an entity with collision enabled to the collision entities vector
 	void AddCollisionEntity(Entity* ent);
 
-
-
+	void AddCollisionComponent(component::CollisionComponent* comp);
+	void RemoveCollisionComponent(component::CollisionComponent* comp);
+	const btDynamicsWorld* GetWorld();
 private:
 	Physics();
 	double m_timeSinceLastColCheck = 0;
 	// How often the collisions are checked
-	const double m_CollisionUpdateInterval = 0.2;
+	const double m_CollisionUpdateInterval = 0.01;
```

Comment:
Does not follow code standard.
Also default values for variables should be in constructor and not header.